plugins/kotlin-serialization/kotlin-serialization-compiler/src/org/jetbrains/kotlinx/serialization/compiler/resolve/NamingConventions.kt

```diff
@@ -68,6 +68,8 @@ object SerialEntityNames {
     const val SERIAL_DESCRIPTOR_CLASS_IMPL = "PluginGeneratedSerialDescriptor"
     const val SERIAL_DESCRIPTOR_FOR_ENUM = "EnumDescriptor"
 
+    const val PLUGIN_UTILS_FILE = "PluginUtils"
```

Comment:
Sorry for late notice, but that just came into my mind: by calling this file 'utils', we risk ending up with long file with a lot of unrelated stuff. I'd suggest avoid such technique and name files using logical groups, e.g. 'PluginExceptions'